### DISCUSSION TITLE
Tycho builds inside eclipse

### DISCUSSION THREAD

**Discussion by rcspirent** [eclipse-tycho/tycho#579] [Q&A]
Is there a convenient way to integrate the tycho build cycle into the development process inside eclipse?
I.e. trigger a build for bundles via m2e but then also add the newly built bundles to an eclipse run-configuration?

**Comment by laeubi** [comment]
I think you must elaborate more here. As one can already import projects into eclipse and build them I don't see whats missing here. And as a source tree can contain numerous executions its a bit hard to guess what should be used as a "runconfig" (in general one uses product definitions for that).

**Comment by rcspirent** [comment]
We have a multi-module tycho (partially) pomless build. We use the CLI commands to build the product.

Now I'm attempting to better integrate this in the usual eclipse developer workflow.
Specifically "build automatically" and launching a product with specified workspace plugins built by tycho and I'm wondering how this would work with pomless tycho builds.
To convert java projects into maven projects seems to require a pom without that the normal java builder activates.

So now I'm a bit puzzled most likely missing some basic information.

**Reply by laeubi** [reply]
Normlay for pomless build one simpyl includes the .project file and it could then be imported in eclipse and started from the product file.
